data-otservbr-global/lib/quests/soul_war.lua

```diff
@@ -1,7 +1,6 @@
 SoulWarQuest = {
 	-- Item ids
 	-- Goshnar's Hatred
-	bagYouDesireItemId = 34109,
```

Comment:
As I said in a previous pull request, this is unnecessary and breaks the encapsulation of the system.

Reply:
As I mentioned in a previous pull request, this proposed change is problematic, and I won’t accept it. Moving a constant, which is an intrinsic part of the quest's logic, outside of the quest file significantly reduces code cohesion. This not only complicates installation for other users, who would have to look for a constant in a non-intuitive location, but also greatly increases the risk of conflicts during operations like cherry-picks or merges.

Keeping the constant within the quest file is precisely to encapsulate its logic, ensuring that all necessary dependencies are present and clearly visible within the context of the quest. Moving this constant to another file breaks this encapsulation and can lead to errors or comprehension difficulties for those attempting to implement or update the quest on their servers.

Without this reversion, I won’t be able to merge your pull request.